profiles/Bylickilabs.json

```diff
@@ -1,6 +1,6 @@
 {
-  "image": "https://user-images.githubusercontent.com/109308073/209451382-c2c90f77-0c2a-45dc-bd06-1f5418910dd9.jpg",
-  "issueId": 1588,
+  "image": "https://user-images.githubusercontent.com/109308073/208296882-ea239594-81ea-419f-ae6f-e3470ccc9a3e.png",
```

Comment:
Please revert the changes in this file. Let us know if you need any help. 🙂

Reply:
Done @Panquesito7 
Thank for helping

Reply:
Hmm, the changes are still there. 🤔